projects/plugins/jetpack/_inc/client/traffic/seo.jsx

```diff
@@ -2,36 +2,116 @@
  * External dependencies
  */
 import React, { Component } from 'react';
-import { __, _x } from '@wordpress/i18n';
+import { connect } from 'react-redux';
+import { __, _x, _n, sprintf } from '@wordpress/i18n';
+import { FacebookPreview, TwitterPreview, SearchPreview } from '@automattic/social-previews';
+import SocialLogo from 'social-logos';
 
 /**
  * Internal dependencies
  */
 import analytics from 'lib/analytics';
-import Card from 'components/card';
 import getRedirectUrl from 'lib/jp-redirect';
 import { withModuleSettingsFormHelpers } from 'components/module-settings/with-module-settings-form-helpers';
 import SettingsCard from 'components/settings-card';
 import SettingsGroup from 'components/settings-group';
 import { ModuleToggle } from 'components/module-toggle';
 import ConnectUserBar from 'components/connect-user-bar';
+import { FormLabel, FormTextarea, FormFieldset } from 'components/forms';
+import FoldableCard from 'components/foldable-card';
+import CustomSeoTitles from './seo/custom-seo-titles.jsx';
+import SimpleNotice from 'components/notice';
+
+export const conflictingSeoPlugins = [
+	'Yoast SEO',
+	'Yoast SEO Premium',
+	'All In One SEO Pack',
+	'All in One SEO Pack Pro',
+];
+
+// Returns first conflicting plugin
+export const getConflictingSeoPlugin = plugins => {
+	const conflictingPlugins = Object.keys( plugins ).filter( plugin => {
+		return plugins[ plugin ].active && conflictingSeoPlugins.includes( plugins[ plugin ].Name );
+	} );
+
+	return plugins[ conflictingPlugins[ 0 ] ];
+};
 
 export const SEO = withModuleSettingsFormHelpers(
 	class extends Component {
+		constants = {
+			frontPageMetaMaxLength: 300,
+			frontPageMetaSuggestedLength: 159,
+			moduleOptionsArray: [ 'advanced_seo_front_page_description', 'advanced_seo_title_formats' ],
+			siteIconPreviewSize: 512,
+		};
+
 		trackConfigureClick = () => {
```

Comment:
Maybe we can remove this now?

Reply:
Good catch! https://github.com/Automattic/jetpack/pull/18810/commits/5debeb6cb1558aca79306b2b5eb6cd353ea037a1